src/jquery.interactive-world-map.js

```diff
@@ -1,34 +1,80 @@
-$(function(){
+;$(function( $, window, document, undefined) {
+
+    var pluginName = "interactiveWorldMap";
+    var marginCountryPopupForLeft;
+    var marginCountryPopupForTop;
+    var dataCountries = {};
+    var countriesFill = {};
+    var colorHighlighted;
+    var patternHover;
+    var colorHover;
+    var pattern2;
+    var pattern;
+    var margins;
```

Comment:
It would be good to understand what these variables should hold. Either rename them so the name is an indication of what they are there for, or at least have brief comments explaining what they are used for.

Best would be both :smiling_imp: